service/src/main/java/datawave/microservice/dictionary/data/DatawaveDataDictionary.java

```diff
@@ -4,40 +4,31 @@
 import datawave.webservice.query.result.metadata.MetadataFieldBase;
 import datawave.webservice.results.datadictionary.DescriptionBase;
 import datawave.webservice.results.datadictionary.DictionaryFieldBase;
-import org.apache.accumulo.core.client.Connector;
-import org.apache.accumulo.core.security.Authorizations;
 
 import java.util.Collection;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.Set;
 
 public interface DatawaveDataDictionary<META extends MetadataFieldBase<META,DESC>,DESC extends DescriptionBase<DESC>,FIELD extends DictionaryFieldBase<FIELD,DESC>> {
-    Collection<META> getFields(String modelName, String modelTableName, String metadataTableName, Collection<String> dataTypeFilters, Connector connector,
-                    Set<Authorizations> auths, int numThreads) throws Exception;
     
-    Map<String,String> getNormalizerMapping();
+    Map<String,String> getNormalizationMap();
     
-    void setNormalizerMapping(Map<String,String> normalizerMapping);
+    void setNormalizationMap(Map<String,String> normalizationMap);
     
-    void setDescription(Connector connector, String metadataTableName, Set<Authorizations> auths, String modelName, String modelTableName, FIELD description)
-                    throws Exception;
+    Collection<META> getFields(ConnectionConfig connectionConfig, Collection<String> dataTypeFilters, int numThreads) throws Exception;
     
-    void setDescription(Connector connector, String metadataTableName, Set<Authorizations> auths, String modelName, String modelTableName, String fieldName,
-                    String datatype, DESC description) throws Exception;
+    void addDescription(ConnectionConfig connectionConfig, FIELD description) throws Exception;
```

Comment:
why addDescription instead of the original setDescription?  We are setting the description for a field which will override any description previously set.  I feel this is more a set than an add.

Reply:
Fixed. I misunderstood how the descriptions were being replaced.